__tests__/app.test.js

```diff
@@ -267,13 +309,29 @@ describe("/api", () => {
           );
         });
     });
-    test("Should return when passed a query that doesn't exist", () => {
+    test("400: Should return when passed a filter query that doesn't exist", () => {
```

Comment:
Again, this would be more readable if it were called category instead of filter.